input/vulnerabilities/QSEE_TrustZone.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name" : "QSEE privilege escalation",
+  "CVE" : [["CVE-2015-6639", "qsee-blog"]],
+  "Coordinated_disclosure" : "true",
+  "Categories" : ["system"],
+  "Details" : [["A vulnerability in the driver for Qualcomm's Trusted Excecution Environment allows code execution in this environment", "qsee-blog"]],
+  "Discovered_by" : [["Gal Beniamini (laginimaineb)", "qsee-blog"]],
+  "Discovered_on" : [["2015-09-27", "qsee-blog"]],
+  "Submission" : [{"by":"dcc52", "on":"2019-07-09"}],
+  "Reported_on" : [["2015-09-27", "qsee-blog"]],
+  "Fixed_on" : [["2015-12-14", "qsee-blog"]],
+  "Fix_released_on" : [["2015-12-14", "qsee-blog"]],
+  "Affected_versions" : [],
```

Comment:
We need some sort of version information if we can get it.

Reply:
Updated - will be there in the next commit